lib/Column/PropertyColumn.php

```diff
@@ -26,7 +29,12 @@ public function __construct(PropertyAccessorInterface $propertyAccessor = null)
     public function createCell(RowData $data, array $options): CellInterface
     {
         $property = $options['property'];
-        $value = $this->accessor->getValue($data->getObject(), $property);
+
+        if (false === $data->isArrayLike()) {
```

Comment:
Add comment .. if the data is an object, then we should operate on the object (the data) rather than the `RowData` object.